src/lib/input/input.ts

```diff
@@ -30,6 +32,9 @@ import {Observable} from 'rxjs/Observable';
 
 const noop = () => {};
 
+const MD_INPUT_SELECTOR = 'md-input';
+const MD_TEXTAREA_SELECTOR = 'md-textarea';
```

Comment:
You can just use the strings inline without introducing constants; the values are obvious and aren't going to change frequently.